web/src/components/space/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react'
+import { useTheme } from 'react-jss'
+
+import useStyles from './styles'
+
+export type SizeTypes =
+  | 'size-half-x'
+  | 'size-1-x'
+  | 'size-1-half-x'
+  | 'size-2-x'
+  | 'size-2-half-x'
+  | 'size-3-x'
+  | 'size-4-x'
+  | 'size-5-x'
+  | 'size-6-x'
```

Comment:
I think we should go with the approach tailwind or theme UI has.

expose just the index.

`size={1}` -> `size-half-x`
`size={2}` -> `size-1-x`
...

Having the "real" value types makes this just more complicated to use!

Reply:
I've already added the "scale" in the branch:
https://github.com/ubergrape/grape-ds/blob/add-input/web/src/styles/scale.ts